addons/mail/static/src/core/common/message.xml

```diff
@@ -53,7 +53,7 @@
                                     (from <a t-att-href="message.resUrl"><t t-esc="message.originThread?.prefix"/><t t-esc="message.originThread?.displayName"/></a>)
                                 </t>
                             </small>
-                            <div t-if="props.message.scheduledDate" t-att-class="{ 'ms-2': props.isInChatWindow and isAlignedRight }" t-att-title="messageService.scheduledDateSimple(props.message)">
+                            <div t-if="props.message.scheduledDate" t-att-class="{ 'ms-2': props.isInChatWindow and isAlignedRight or (!isOriginThread and !message.is_transient) }" t-att-title="messageService.scheduledDateSimple(props.message)">
```

Comment:
We will likely forget to adapt this condition is we ever adapt the other one. Maybe there should be a common getter for `hasThreadName` (or something like that).

Reply:
Hello @seb-odoo

Thanks for your feedback 😃 since `hasThreadName` was removed in saas-16.3 I checked for alternatives but didn't find any alternative so I created a new method.